frontend/shoot-client/src/components/Game/Game.context.tsx

```diff
@@ -57,17 +67,27 @@ export const GameProvider: React.FC = ({ children }) => {
                 if (notification.hasScores()) {
                     console.log('score update');
                     // Handle A Score Update
-                    let updateState = {...state};
-                    updateState.score[0] = notification.getScores()?.getTeam1() as number;
-                    updateState.score[1] = notification.getScores()?.getTeam2() as number;
-                    setState(updateState);
+                    let { score }  = { ...state };
```

Comment:
The lets seem a little strange not sure why you have to pull it out of state just to override it. Could all be done in a single line like this

```suggestion
                    setState({...state, score: [ notification.getScores()?.getTeam1() as number, notification.getScores()?.getTeam2() as number ]});
```